src/schemas/components/SchemaFormPropertyArray.vue

```diff
@@ -21,6 +21,11 @@
         />
       </template>
     </p-draggable-list>
+    <template v-if="!allowCreate && property.maxItems">
```

Comment:
nit: probably nbd but if for some reason `property.maxItems === 0`, you wouldn't be able to add anything and you wouldn't ever see the message